From b21b9cd9cb6a08b9bfc9d025b4e6a64a12e50d65 Mon Sep 17 00:00:00 2001
From: Daniel-Constantin Mierla <miconda@gmail.com>
Date: Fri, 8 Apr 2016 15:10:55 +0200
Subject: [PATCH] Makefile.defs: re-enable mem join define

- lost when introduced the option to select the memory manager by
  command line parameter -x

(cherry picked from commit 33c6aafcd46269ce283908f49fbf70b9d73624b1)
---
 Makefile.defs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.defs b/Makefile.defs
index 26205e8..245f9b4 100644
--- a/Makefile.defs
+++ b/Makefile.defs
@@ -664,6 +664,7 @@ C_DEFS= $(extra_defs) \
 # debugging symbols in all cases (-g). --andrei
 
 # memory managers and related debug mode
+C_DEFS+= -DMEM_JOIN_FREE
 # enable f_malloc
 C_DEFS+= -DF_MALLOC
 # enable q_malloc
-- 
2.8.0.rc3

